games-engines/openmw/openmw-9999.ebuild

```diff
@@ -96,6 +96,7 @@ src_configure() {
 		-DMORROWIND_DATA_FILES="${EPREFIX}/usr/share/morrowind-data"
 		-DUSE_SYSTEM_TINYXML=ON
 		-DDESIRED_QT_VERSION=5
+		-DBULLET_USE_DOUBLES=ON
 	)
```

Comment:
Think we can add a USE flag to flip this variable? I'm not sure if it might result in a subtle behavior change.

Reply:
It will result in behavior change, yes. https://github.com/OpenMW/openmw/pull/2049 has a long discussion about it. Yes, we can move the choice of "buggy" vs "correct" to the user, though I don't see much benefit in such USE flag on openmw side.

Reply:
I just thought about the case where the USE flag can be useful: if some other program depends on bullet, but requires single precision instead, then user can't install both that program and openmw together, if there is no such USE flag in openmw.

I don't know how important it is though, and whether there are such programs.

Reply:
I don't think I have the time enumerating all packages that depends on bullet and examine them one by one, but since float-point computing is very dependent on the precision bits, I'd suggest you test the 9999 version with different bullet precisions and make sure it doesn't end up dropping FPS rate dramatically or having other performance impacts.